test/bigtest/tests/user-proxy-edit-test.js

```diff
@@ -0,0 +1,100 @@
+import {
+  beforeEach,
+  describe,
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import findUser from '@folio/plugin-find-user';
+import FindUserInteractor from '@folio/plugin-find-user/test/bigtest/interactors/findUser';
```

Comment:
I _love_ that you wrote these unit tests for managing proxies ... but I don't think ui-users has a direct dependency on `@folio/plugin-find-user` so I'm not sure about this :/ There is [a user-proxy integration test](https://github.com/folio-org/platform-core/blob/9423464f9720330b0494178f870a86d837d5d52d/test/ui-testing/new-proxy.js) in platform-core that handles proxy management. 

What do you think?